src/service_discovery/mod.rs

```diff
@@ -287,7 +289,7 @@ mod tests {
             unwrap!(
                 el1.send(CoreMessage::new(move |core, poll| {
                     unwrap!(
-                        ServiceDiscovery::start(core, poll, listeners_1, token_1, 65_530),
+                        ServiceDiscovery::start(core, poll, listeners_1, token_1, 0xFFFA),
```

Comment:
this is horrible - even if clippy is complaining i would suppress that warning in these places and leave them as is.